cookbooks/aws-parallelcluster-computefleet/recipes/install/parallelcluster_node.rb

```diff
@@ -34,7 +34,9 @@
 end
 
 if aws_region.start_with?("us-iso") && !is_custom_node?
-  node.default['cluster']['custom_node_package'] = "#{node['cluster']['artifacts_s3_url']}/dependencies/node/aws-parallelcluster-node.tgz"
+  node_package = "aws-parallelcluster-node-#{node['cluster']['parallelcluster-node-version']}.tgz"
+
+  node.default['cluster']['custom_node_package'] = "#{node['cluster']['s3_url']}/parallelcluster/#{node['cluster']['parallelcluster-node-version']}/node/#{node_package}"
```

Comment:
Are we making changes in the S3 location to upload default Node package in this location. As I dont see node package in s3 for other regions?

Reply:
when we release in adc, we will also have to release a node and cookbook package so that they can use those for build image. I think we have a similar path where we have been publish the cookbook in s3 in commercial.